### PR TITLE
EVG-19274 Downlevel ECS Pod LatestStatusInfo failures to warning level

### PR DESCRIPTION
**Pull request by hadjri**
EVG-19274

### Description
Temporarily downleveling this to reduce noise when filtering `metadata.level >= 70` in staging since we're aware of this issue.

### Testing
N/A

### Documentation
remember to add or edit docs in the docs/ directory if relevant

### CODE REVIEW

File: units/pod_health_check.go [hunk 1]
```diff
@@ -119,7 +119,14 @@ func (j *podHealthCheckJob) Run(ctx context.Context) {
...
 	info, err := j.ecsPod.LatestStatusInfo(ctx)
 	if err != nil {
-		j.AddError(errors.Wrap(err, "getting cloud pod's status info"))
+		grip.Warning(message.Fields{
+			"message":           "unable to get cloud pod's status info",
+			"pod":               j.PodID,
+			"status":            j.pod.Status,
+			"last_communicated": j.pod.TimeInfo.LastCommunicated,
+			"job":               j.ID(),
+			"err":               err.Error(),
```
Comment (Kimchelly) [inline units/pod_health_check.go:128]: `message.WrapError` instead of making a `err` field.

**Review by Kimchelly — APPROVED** [review]
LGTM mod making a ticket to reset it back to error and one trivial comment.

**Comment by hadjri** [comment]
> "last_communicated": j.pod.TimeInfo.LastCommunicated,

https://jira.mongodb.org/browse/EVG-19284

**Comment by hadjri** [comment]
https://jira.mongodb.org/browse/EVG-19284

**Pull request merged**